kotlin/workflow-core/src/main/java/com/squareup/workflow/StatefulWorkflow.kt

```diff
@@ -71,10 +73,15 @@ abstract class StatefulWorkflow<
    * If the workflow is being restored from a [Snapshot], [snapshot] will be the last value
    * returned from [snapshotState], and implementations that return something other than
    * [Snapshot.EMPTY] should create their initial state by parsing their snapshot.
+   * @param scope
+   * The [CoroutineScope] in which this workflow lives. The scope will be cancelled when the
```

Comment:
Can we provide an example of how to use it?

Reply:
I'm not sure what kind of example would be useful. It's basically only there to support `Worker`s (which users don't need to care about unless they want to look at source directly), or cancellation of some external resource, and I don't even know yet what the best way will be to use this scope to cancel those. I'll file an issue to track improving this doc as we go forward: #290.